doc/md_src_files/060_decision-trees.md

```diff
@@ -89,7 +89,7 @@ Unfortunately, it also makes it a bit too easy to overcomplicate the decision.
 
 The SSVC version 1  ~applier~ deployer tree had 225 rows when we wrote it out in long text form.
 It only has four outcomes to differentiate between.
-Thus on average that decision process treats one situation (combination of decision values) as equivalent to 65 other situations.
+Thus, on average that decision process treats one situation (combination of decision values) as equivalent to 65 other situations.
```

Comment:
Does this need a second comma too? i.e., "Thus, on average, ..."

Reply:
nope. one or the other